deploy/helm/kuberhealthy/templates/khcheck-network-connection.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.check.networkConnection.enabled }}
+---
+apiVersion: comcast.github.io/v1
+kind: KuberhealthyCheck
+metadata:
+  name: network-connection-check
+  namespace: {{ .Release.Namespace }}
+spec:
+  runInterval:  {{ .Values.check.networkConnection.runInterval }}
+  timeout: {{ .Values.check.networkConnection.timeout }}
+  podSpec:
+    containers:
+    - name: deployment
+      image: {{ .Values.check.networkConnection.image.repository }}:{{ .Values.check.networkConnection.image.tag }}
+      imagePullPolicy: IfNotPresent
+      env:
+        - name: CHECK_TIME_LIMIT
+          value: {{ .Values.check.networkConnection.timeout }}
+{{- range $key, $value := .Values.check.networkConnection.extraEnvs }}
+        - name: {{ $key }}
+          value: {{ $value | quote }}
+{{- end }}
+      resources:
+        requests:
+          memory: 5Mi
```

Comment:
```suggestion
          memory: 5Mi
        {{- if .Values.securityContext.enabled }}
        securityContext:
          allowPrivilegeEscalation: {{ .Values.securityContext.allowPrivilegeEscalation }}
          readOnlyRootFilesystem: {{ .Values.securityContext.readOnlyRootFilesystem }}
        {{- end}}
```

Reply:
from my point of view we don't need the `securityContext` here.
* the application itself doesn't need any extended permissions (only does a tcp/udp connection check, no additional file operations).
* container is build with a numeric user and point to `nobody` (https://github.com/Comcast/kuberhealthy/pull/443/files#diff-b80ba549f0d3f00bfc542d54b0e03f7cR10) 
* if a `psp` is deployed where the usage of `root` is forbidden, kubernetes can schedule pods without `securityContext` when the `USER` field is a numeric value.